e2e/playwright/list-views/src/tests/trash.spec.ts

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, LoginPage, NodesApi, SitesApi, Utils, test } from '@alfresco/playwright-shared';
+import { Site } from '@alfresco/js-api';
+
+test.describe('Trash', () => {
+  const apiClientFactory = new ApiClientFactory();
+  let nodesApi: NodesApi;
+  let siteActionsAdmin: SitesApi;
+
+  const username = `user-${Utils.random()}`;
+  const siteName = `site-${Utils.random()}`;
+  const fileSite = `file1-${Utils.random()}.txt`;
+
+  const folderUser = `folder-${Utils.random()}`;
+  let folderUserId: string;
+  const fileUser = `file-${Utils.random()}.txt`;
+  let fileUserId: string;
```

Comment:
please move fileUserId to beforeAll and make it const

Reply:
in next commit deleting this variable as not used